### DISCUSSION TITLE
RustyKeys コミュニティにようこそ！

### DISCUSSION THREAD

**Discussion by KOBA789** [KOBA789/rusty-keys#2] [Announcements]
## 👋 RustyKeys コミュニティにようこそ！
これは RustyKeys に関する情報交換や交流を目的としたコミュニティです。

たとえば、このような使い方があるでしょう:
- ❓ 組み立てやファームウェア開発に関する質問をする
- 💡 あなたのオリジナルのファームウェアのアイデアを披露する
- 🔧 おすすめの開発環境の情報を共有する

RustyKeys コミュニティはまだ生まれたばかりです。
楽しく有意義な場所にするにはみなさんの協力が必要です！💪

もしよければ自己紹介をお願いします。

---
はじめましての人ははじめまして。KOBA789 です。RustyKeys の作者です。
趣味でも仕事でも Rust を書いており、組込み用途にも Rust を使えないかと模索中です。
一方で自作キーボードの経験はあまりありません。キースイッチはロープロファイルで軽いものが好きですが、キーキャップのバリエーションが少ないのが悩みです。よいキーキャップを知っている人がいたらぜひ教えてください。
これからよろしくお願いします！

**Comment by mtei** [comment]
はじめまして、Rust も Raspberry Pi Pico も未経験ですがよろしくお願いします。
遊舎工房で、RustyKeys がアナウンス翌日には売り切れてて、出遅れて残念です。
もしも Raspberry Pi Pico の入荷がネックになっているのなら、それなしの構成でも出してもらえると嬉しいです。既にRaspberry Pi Picoは買い置きがあるので。

**Comment by kk0kojirr** [comment]
連休中にrdbを一年遅れて作ろう！としていたら、何やら今年も楽しそうなRustプロジェクトがはじまろうとしていたのを見つけて、初回版を逃しつつ、焦ってキースイッチ、キーキャップなしで駆けつけました。なんだか、毎朝まで準備されているのを目にすると、こちらも目一杯いじり倒そうという感じです！
